asl-translator.cabal

```diff
@@ -22,7 +22,7 @@ flag unsafe-operations
 
 flag asl-lite
   Description: Use a trimmed-down set of instructions for the ASL specification (sufficient to disassemble u-boot).
-  Default: False
+  Default: True
```

Comment:
This should be a separate commit if we do it - it would require a coordinating change with dismantle

Reply:
Oops -- I think that is a mistake.

Reply:
I've changed it back to False